src/main/java/uk/gov/justice/digital/zone/RawZone.java

```diff
@@ -0,0 +1,75 @@
+package uk.gov.justice.digital.zone;
+
+import org.apache.spark.api.java.JavaRDD;
+import org.apache.spark.sql.Dataset;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SaveMode;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.types.DataTypes;
+import org.apache.spark.sql.types.StructType;
+import uk.gov.justice.digital.config.JobParameters;
+import uk.gov.justice.digital.util.SourceReference;
+
+import javax.inject.Inject;
+import javax.inject.Singleton;
+import java.util.List;
+
+import static org.apache.spark.sql.functions.*;
+import static org.apache.spark.sql.functions.col;
+
+@Singleton
+public class RawZone implements Zone {
+
+    private static String DELTA_FORMAT = "delta";
+    private String rawPath;
+
+    @Inject
+    public RawZone(JobParameters jobParameters){
+        this.rawPath = jobParameters.getRawPath();
+    }
+
+    @Override
+    public Dataset<Row> process(JavaRDD<Row> rowRDD, SparkSession spark) {
```

Comment:
Consider making this void if we're not using the result.

Reply:
I think that Raw DataFrame is going to use by next step like Structured, Curated.